src/app.py

```diff
@@ -170,6 +171,74 @@ def bootstrap(_return=0):
         s.communicate()
 
 
+def send_user_usage(target_host):
+    """Send user usage to Google Analytics."""
+
+    try:
+        ga_endpoint = 'https://www.google-analytics.com/collect'
+        ga_tracking_id = 'UA-110383676-1'
+        send_kpi = convert_str_to_bool(os.getenv('SEND_ANONYMOUS_USAGE_INFO', str(False)))
+
+        if send_kpi:
```

Comment:
if you conditionally invoke send_user_usage() in line 31 or do something like: "if not send_kpi: return" it would allow to shift the whole block 1 position to the left